org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoProductWindow.java

```diff
@@ -443,9 +443,16 @@ protected int getSelectedWarehouseId() {
 	@Override
 	protected void initParameters() {
 		int M_Warehouse_ID = Env.getContextAsInt(Env.getCtx(), p_WindowNo, "M_Warehouse_ID");
-		int M_PriceList_ID = Env.getContextAsInt(Env.getCtx(), p_WindowNo, "M_PriceList_ID");
 		
-		int M_PriceList_Version_ID = findPLV (M_PriceList_ID);
+		String ignorePriceList = Env.getContext(Env.getCtx(), p_WindowNo,
```

Comment:
Checking the occurrences of M_Product_ID field, I tend to think the majority of these fields are not related to the default price list.
I can see about 70 editable M_Product_ID fields like:
* Accounting Schema > Account Schema Dimensions
* Cash Plan > Cash Plan Line
* Commission > Commission Line
* Cost Adjustment > Cost Adjustment Line
* Customer RMA > RMA Line
* GL Distribution > Distribution
* GL Distribution > Line
* GL Journal > Line
* Import GL Journal > General Ledger
* and a long etcetera

Because there are more cases where price list must be ignored, I think the variable must be named the opposite UsePriceListInProductInfo (instead of Ignore) - and the code checks the opposite (that UsePriceListInProductInfo=Y)

In this case, instead of changing the 4 windows:
* Physical Inventory
* Price List
* Inventory Move
* Internal Use Inventory
as in the migration script

It would be the opposite, just windows that must USE the price list will have the predefined context variable, these would be:
* Sales Order
* Purchase Order
* Import Order
* Invoice Vendor
* Invoice Customer
* Import Invoice
* Requisition

I think there are not more than these 7, but if found any other that requires it, is easy to just add that.

Reply:
Hi @CarlosRuiz-globalqss ,
An minor issue that arise from this is it does break backward compatibility - I will have to add that to my custom sales order ad_window definition or it will break the product lookup that's working prior to this change.

Reply:
that's correct - in such case the update must not be directly to the ad_window_id - but we can do the update to any window containing tabs for C_Order, C_Invoice, I_Order, I_Invoice and M_Requisition

Reply:
@CarlosRuiz-globalqss took notes, Let me change and submit new PR.